packages/build/config/tsconfig.common.json

```diff
@@ -19,6 +19,7 @@
     "target": "es2017",
     "sourceMap": true,
     "declaration": true,
+    "declarationMap": true,
```

Comment:
Is this necessary? I thought that TypeScript automatically enables declaration maps when project references are used.

Also I think declaration maps are no longer needed for IDEs, see https://github.com/microsoft/TypeScript/pull/32028

Reply:
Reverted.

Reply:
I'm not 100% sure if it's fine. See https://www.typescriptlang.org/docs/handbook/project-references.html#declarationmaps. I'll test.